From cb88d114950a98ce43a932d3f78f34fc7f075fc2 Mon Sep 17 00:00:00 2001
From: "Wood, Brian J" <brian.j.wood@intel.com>
Date: Thu, 21 Mar 2019 14:14:22 -0700
Subject: [PATCH 1336/1418] drm/i915: Removing dead_code from function block

There was found to be code from previous commit that's
considered dead_code as during execution could never be
reached. We're cleaning that up with this commit


Change-Id: Ib6ca39dea9b5554f139447855c4ad248d0663d7d
Signed-off-by: Wood, Brian J <brian.j.wood@intel.com>
---
 drivers/gpu/drm/i915/intel_initial_modeset.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_initial_modeset.c b/drivers/gpu/drm/i915/intel_initial_modeset.c
index 4a3a6e7..74630533 100644
--- a/drivers/gpu/drm/i915/intel_initial_modeset.c
+++ b/drivers/gpu/drm/i915/intel_initial_modeset.c
@@ -241,7 +241,6 @@ static int update_atomic_state(struct drm_device *dev,
 			       struct drm_connector *connector,
 			       struct drm_display_mode *mode)
 {
-	struct drm_framebuffer *fb = NULL;
 	struct drm_crtc *crtc;
 	int ret;
 
@@ -259,13 +258,6 @@ static int update_atomic_state(struct drm_device *dev,
 	if (ret)
 		return ret;
 
-	/* Set up primary plane if a framebuffer is allocated */
-	if (fb) {
-		ret = update_primary_plane_state(state, crtc, mode, fb);
-		if (ret)
-			return ret;
-	}
-
 	return 0;
 }
 
-- 
2.7.4

